src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -73,8 +73,8 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
             //   doing a "dance" while fighting
             if (owner->IsPet() && i_target->GetTypeId() == TYPEID_PLAYER)
             {
-                dist = i_target->GetCombatReach();
-                size = i_target->GetCombatReach() - i_target->GetObjectSize();
+                dist = 1.0f; //i_target->GetCombatReach();
```

Comment:
I thought the original values could be variable since the checks for `GetCombatReach()` and `GetObjectSize()` were used. I don't know if the results could be a fixed value. Maybe @MrSmite knows more about it, since he wrote it.